src/config/editorsImgMap.ts

```diff
@@ -25,15 +23,23 @@ interface EditorImageMap {
 }
 
 const imgMap: EditorImageMap = {
-  [ANDROIDSTUDIO]: androidStudio128Path,
-  [ATOM]: atom128Path,
-  [CHROME]: chrome128Path,
-  [ECLIPSE]: eclipse128Path,
-  [SUBLIMETEXT2]: sublimeText128Path,
-  [SUBLIMETEXT3]: sublimeText128Path, // Should this be a different image?
-  [VIM]: vim128Path,
-  [VSCODE]: vsCode128Path,
-  [XCODE]: xcode128Path
+  [new editors.AndroidStudio().name]: androidStudio128Path,
```

Comment:
Shouldn't we just do a list of uninstantiated editor classes? Then we can instantiate them when using them.